lua/lspconfig/server_configurations/gitlab_ci_ls.lua

```diff
@@ -0,0 +1,33 @@
+local util = require 'lspconfig.util'
+
+local home = util.path.join(vim.loop.os_homedir(), '.cache/gitlab-ci-ls/')
```

Comment:
for clear.. could you rename to cache_dir.

Reply:
Wops, I'm a 🥔 .
Fixed.